pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -235,13 +235,9 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1alpha1.PipelineRun) er
 	pipelineMeta, pipelineSpec, err := resources.GetPipelineData(pr, getPipelineFunc)
 	if err != nil {
 		c.Logger.Errorf("Failed to determine Pipeline spec to use for pipelinerun %s: %v", pr.Name, err)
-		pr.Status.SetCondition(&apis.Condition{
-			Type:   apis.ConditionSucceeded,
-			Status: corev1.ConditionFalse,
-			Reason: ReasonCouldntGetPipeline,
-			Message: fmt.Sprintf("Error retrieving pipeline for pipelinerun %s: %s",
-				fmt.Sprintf("%s/%s", pr.Namespace, pr.Name), err),
-		})
+		pr.Fail(ReasonCouldntGetPipeline,
+			fmt.Sprintf("Error retrieving pipeline for pipelinerun %s: %s",
```

Comment:
This appears to be a `fmt.Sprintf` as an arg to `fmt.Sprintf`, which is hard to read. Can you pass in `pr.Namespace` and `pr.Name` and assemble them in the format string?

(and some below)

Reply:
I intentionally didn't modify the existing pattern (fmt in fmt) to simplify review, but I'm happy to do so.